packages/core/src/notifications/activation.ts

```diff
@@ -13,7 +13,7 @@ import { AuthState } from './types'
 import { getLogger } from '../shared/logger/logger'
 
 /** Time in MS to poll for emergency notifications */
-const emergencyPollTime = 1000 * 10 * 60
+const emergencyPollTime = 1000 * 60 * 10
```

Comment:
I added in some constants for time in `datetime.ts`, we should look to start using these